vm/rust/src/lib.rs

```diff
@@ -222,14 +216,13 @@ fn build_block_context(
     chain_id_str: &str,
     block_number: c_ulonglong,
     block_timestamp: c_ulonglong,
-    sequencer_address: StarkFelt,
 ) -> BlockContext {
     BlockContext {
         chain_id: ChainId(chain_id_str.into()),
         block_number: BlockNumber(block_number),
         block_timestamp: BlockTimestamp(block_timestamp),
 
-        sequencer_address: ContractAddress(PatriciaKey::try_from(sequencer_address).unwrap()),
+        sequencer_address: ContractAddress::default(),
```

Comment:
> Sequencer address is unnecessary for execution 

it actually is, fee contract asserts that `sequencer_address != 0x0`

We can use the fallback sequencer address, see https://github.com/NethermindEth/juno/pull/896#discussion_r1270767340

Reply:
Interesting. I wonder why it worked with my manual tests.